### ISSUE TITLE
Status of the project

### ISSUE THREAD

**Issue by TobiasNeuhauser** [TES3MP/openmw-tes3mp#587]
Hi David,

can we kindly have a status update of the project. I mean open source project like this owes us absolutely nothing. But to save time and curiosity on both sides, it would be nice to see where this very interesting multiplayer/coop project is heading to. I mean if this project is on ice for the time being, I am fine with it, then I would not spending time on check this website once in a while and getting on your nerve, instead I would rather look for other games or mods. Will there be a tes3mp release based on OMW 0.47 soon or will it merge with OMW? Keep up the great work or let us know what is going on with tes3mp. Thanks a lot. Sorry for bothering you.

**Comment by datatypevoid** [comment]
The README states it is using `0.47.0`, for the record.

**Comment by SourceCodeDeleted** [comment]
The code is up to this version but there is no release candidate .

The latest release if from: 
TES3MP 0.7-alpha
@davidcernat davidcernat released this on Oct 9, 2018

**Comment by TobiasNeuhauser** [comment]
Hey SourceCodeDeleted,

Yepp you are right. The release candidate is based on OMW 0.44. Although it works fine, the client side script synchronization is not fully implemented. So to finish Morrowind with a buddy, you sometime have to use the console to trigger or progress in a quest. But quest mods like "Tel Meskoa and Tel Matouigius" or the "Wizard Island" requiring to sleep to pass time, do not work at all for me.

**Comment by datatypevoid** [comment]
Well just build the latest then, it works fine on linux.

**Comment by datatypevoid** [comment]
> ...
> Although it works fine, the client side script synchronization is not fully implemented. So to finish Morrowind with a buddy, you sometime have to use the console to trigger or progress in a quest. But quest mods like "Tel Meskoa and Tel Matouigius" or the "Wizard Island" requiring to sleep to pass time, do not work at all for me.

Also this has nothing to do with anything anyone said in this post and at best is just a rehash of what has been repeated painstakingly over and over again here on github, on the discord, and also on the steam forums.

**Comment by datatypevoid** [comment]
Lol then build the latest if you they don't have a release, it's considered more secure to build you own software anyway.

**Comment by TobiasNeuhauser** [comment]
I am not into building from source code, otherwise I would not have posted here. And even if I wanted to build tes3mp from source, first a proper release candidate is needed. Building from a snapshot of the github code updated to August 2020, no thanks.

**Comment by slowpokefarm** [comment]
I suppose this thread is initially about the overall status of the project and not about building from sources, isn't it?

**Comment by datatypevoid** [comment]
You are absolutely right but the whiny passive-aggressive "I will leave you if you don't do something" attitude is poor to say the least, and is something that I have seen in various projects cause some/many developers want to contribute even less than they might already. The whole thing could have been worded much differently imo. Just an opinion, don't like it move on.

**Comment by TobiasNeuhauser** [comment]
Hi Slowpokefarm,

Correct. Jut want to know the status about tes3mp. 

@datatypevoid 
whiny attitude, are yoou kidding me? I just asked politely about the status. You want me to be rude or what? And when a project or games is halted, then people normally go on, or what should we do otherwise? Please tell me what we should do? Still checking a website of a dead project and further getting the ex developer for that on the nerve. If there would be a slightly more revealing status like Daggerfall Unity is doing, then I would not leave here any post. That is not passive aggressiveness, rather it is "wanting to know what is going on". If asking for status of project pisses you or developers off or you feel politely threatened, than be pissed off or threatened, I don't care. That was never the intention of my post, so please don't over interpret here anything.

**Comment by slowpokefarm** [comment]
Two wrongs don't make a right sadly, but I understand your point and completely agree about the attitude and threatening.
I also didn't intend to be aggressive about the topic of this issue, but there's literally a separate issue where you guys discuss building from latest sources, which is also great.

But the status of the project is definitely more concerning at this point. 

David has a Patreon page https://www.patreon.com/davidcernat, where I would gladly join as a patron, but the lack of releases or news or other kinds of feedback really keeps me away from doing that. I mean, we don't even know if the author has survived the pandemic etc.

**Comment by SourceCodeDeleted** [comment]
Good news,

I have built the source as I understand.  I will post to the following thread. 
https://github.com/TES3MP/openmw-tes3mp/issues/586

**Comment by jotoho** [comment]
While the latest commit on the default branch `0.7.1` is dated August 2020
that doesn't mean there hasn't been any activity since then.

@davidcernat has last merged 46 commits into the default branch on May 2nd
in PR #581. The commits are just dated as older than that PR within the commit
metadata.

**Comment by chemputer** [comment]
> While the latest commit on the default branch `0.7.1` is dated August 2020
> that doesn't mean there hasn't been any activity since then.
> 
> @davidcernat has last merged 46 commits into the default branch on May 2nd
> in PR #581. The commits are just dated as older than that PR within the commit
> metadata.
> 

If only more people knew how to actually _use_ git beyond the absolute basics. 

Thanks for sharing that with the masses.
